_includes/headtags.html

```diff
@@ -1,14 +1,12 @@
 <meta charset="utf-8">
 <!-- Canonical and Alternate Links -->
+<link rel="canonical" hreflang="en" href="{{ site.url }}{{ page.url }}">
 {% assign sorted_languages = site.languages | sort %}
 {% for lng in sorted_languages %}
```

Comment:
Can these be done in the one line?
```suggestion
{% for lng in site.languages | sort %}
```

Reply:
I think so - will check.